pkg/model/gcemodel/api_loadbalancer.go

```diff
@@ -93,8 +97,17 @@ func createPublicLB(b *APILoadBalancerBuilder, c *fi.ModelBuilderContext) error
 			Network:      network,
 			SourceRanges: b.Cluster.Spec.API.Access,
 			TargetTags:   []string{b.GCETagForRole(kops.InstanceGroupRoleControlPlane)},
-			Allowed:      []string{"tcp:443"},
+			Allowed:      []string{"tcp:" + strconv.Itoa(wellknownports.KubeAPIServer)},
 		})
+		if b.Cluster.UsesNoneDNS() {
```

Comment:
Do we need this?  This would be for e.g. humans running `kubectl`?

Reply:
Not quite pretty, but with Public LB, the nodes need access to it and this was the best idea at the time except for allowing all traffic.